src/services/BadgesService.ts

```diff
@@ -48,6 +55,10 @@ export class BadgesService {
     return this.createBadgesList(otterspaceBadges)
   }
 
+  static async getBadgesByCid(badgeCid: string) {
```

Comment:
I think this should be `badgeSpecCid` @1emu?

Reply:
yes!